checkov/common/runners/object_runner.py

```diff
@@ -205,7 +205,7 @@ def _get_jobs(self, definition: dict[str, Any]) -> dict[int, str]:
                     end_line: int = job_instance.get(END_LINE, -1)
                     end_line_to_job_name_dict[end_line] = job_name
 
-                    steps = job_instance.get('steps')
+                    steps = [step for step in job_instance.get('steps', []) if step]
```

Comment:
Nice! Just need to make sure steps cannot be defined explicitly as None inside job_instance, since in that case it can crash.

Reply:
just use `[step for step in job_instance.get('steps') or [] if step]` this covers also the explicit `None` case 🙂